pkg/kapp/config/conf.go

```diff
@@ -122,11 +122,14 @@ func (c Conf) WaitRules() []WaitRule {
 	return rules
 }
 
-func (c Conf) LabelScopingMods() func(kvs map[string]string) []ctlres.StringMapAppendMod {
+func (c Conf) LabelScopingMods(disableDefault bool) func(kvs map[string]string) []ctlres.StringMapAppendMod {
 	return func(kvs map[string]string) []ctlres.StringMapAppendMod {
 		var mods []ctlres.StringMapAppendMod
 		for _, config := range c.configs {
 			for _, rule := range config.LabelScopingRules {
+				if disableDefault && rule.IsDefault {
```

Comment:
nit: Change name to `disableDefaultRules` instead of `disableDefault`

Reply:
Sure 😃